tests/unit/test_package.py

```diff
@@ -13,362 +15,192 @@ def mock_swagger_generator():
 
 
-def test_can_create_app_packager_with_no_autogen():
-    # We can't actually observe a change here, but we want
-    # to make sure the function can handle this param being
-    # False.
-    config = Config.create(autogen_policy=False)
-    packager = package.create_app_packager(config)
-    assert isinstance(packager, package.AppPackager)
-
-
-def test_preconfigured_policy_proxies():
-    policy_gen = mock.Mock(spec=ApplicationPolicyHandler)
-    config = Config.create(project_dir='project_dir', autogen_policy=False)
-    policy_gen.generate_policy_from_app_source.return_value = {
-        'policy': True}
-    policy = policy_gen.generate_policy_from_app_source(config)
-    assert policy == {'policy': True}
-
-
-def test_sam_generates_sam_template_basic(sample_app,
-                                          mock_swagger_generator,
-                                          mock_policy_generator):
-    p = package.SAMTemplateGenerator(mock_swagger_generator,
-                                     mock_policy_generator)
-    config = Config.create(chalice_app=sample_app,
-                           api_gateway_stage='dev')
-    template = p.generate_sam_template(config, 'code-uri')
-    # Verify the basic structure is in place.  The specific parts
-    # are validated in other tests.
-    assert template['AWSTemplateFormatVersion'] == '2010-09-09'
-    assert template['Transform'] == 'AWS::Serverless-2016-10-31'
-    assert 'Outputs' in template
-    assert 'Resources' in template
-
-
-def test_sam_injects_policy(sample_app,
-                            mock_swagger_generator,
-                            mock_policy_generator):
-    p = package.SAMTemplateGenerator(mock_swagger_generator,
-                                     mock_policy_generator)
-
-    mock_policy_generator.generate_policy_from_app_source.return_value = {
-        'iam': 'policy',
-    }
-    config = Config.create(chalice_app=sample_app,
-                           api_gateway_stage='dev')
-    template = p.generate_sam_template(config)
-    assert template['Resources']['APIHandler']['Properties']['Policies'] == [{
-        'iam': 'policy',
-    }]
-    assert 'Role' not in template['Resources']['APIHandler']['Properties']
-
-
-def test_sam_injects_swagger_doc(sample_app,
-                                 mock_swagger_generator,
-                                 mock_policy_generator):
-    p = package.SAMTemplateGenerator(mock_swagger_generator,
-                                     mock_policy_generator)
-    mock_swagger_generator.generate_swagger.return_value = {
-        'swagger': 'document'
-    }
-    config = Config.create(chalice_app=sample_app,
-                           api_gateway_stage='dev')
-    template = p.generate_sam_template(config)
-    properties = template['Resources']['RestAPI']['Properties']
-    assert properties['DefinitionBody'] == {'swagger': 'document'}
-
-
-def test_can_inject_environment_vars(sample_app,
```

Comment:
Looks like we missed this one when porting over the tests. I also cannot find it in the ported over implementation.

Reply:
Good catch.  Updated.